src/scss/_objects.modules.scss

```diff
@@ -90,6 +90,18 @@
   }
 }
 
+.accordion-cta-image {
```

Comment:
As @phuson mentioned above, you should be using BEM for the class names.  Also, it would be great if you could pull out the Accordion SCSS into its own file, since ultimately these modules will be pulled into core Grommet, and having separate SCSS files will make the transition much easier.  Lastly, please refer to Grommet core for the conventions they use for SCSS, both in terms of naming and class nesting.